packages/react/src/AvatarStack/AvatarStack.tsx

```diff
@@ -307,16 +334,44 @@ const AvatarStack = ({
     sxProp as SxProp,
   )
 
-  return (
-    <AvatarStackWrapper count={count} className={wrapperClassNames} sx={avatarStackSx}>
+  const AvatarStackBody = ({children}: React.ComponentPropsWithoutRef<'div'>) => {
```

Comment:
If we're making a new component as a helper here we should make sure it's defined outside of `AvatarStack` otherwise we'll run into an issue with React not being able to reconcile `AvatarStackBody` since it's being re-created every render.